sources/dataset.py

```diff
@@ -426,16 +430,19 @@ def draw_image(self, image_id, datapath):
 
         image = np.ones([self.image_size, self.image_size, 3], dtype=np.uint8)
         image = image * np.array(image_info["background"], dtype=np.uint8)
+        labelled_image = np.ones([self.image_size, self.image_size, 3], dtype=np.uint8) * 255
 
         # Get the center x, y and the size s
         if image_info["labels"][0]:
             color, x, y, s = image_info["shape_specs"][0]
             color = tuple(map(int, color))
             image = cv2.rectangle(image, (x - s, y - s), (x + s, y + s), color, -1)
+            labelled_image = cv2.rectangle(labelled_image, (x - s, y - s), (x + s, y + s), self.class_info[0]["color"], -1)
         if image_info["labels"][1]:
             color, x, y, s = image_info["shape_specs"][1]
             color = tuple(map(int, color))
```

Comment:
Is `color` stored as string? I wonder why you have to cast the values of `color` to int.

Reply:
It guess `np.int` are involved. That was a kind of mystery for me: `cv2` method seems not compatible with such a format, hence the ugly casting.